### ISSUE TITLE
[Design] Update color(grey-40)

### ISSUE THREAD

**Issue by joebell93** [sky-uk/toolkit#185]
Moved from https://github.com/sky-uk/toolkit-core/issues/166 to ease with **Toolkit 2.0** monorepo tracking

---

>We need to change one of the greys in the colour palette as it turns out it fails AA accessibility for normal size text.
>We need to change **#7f7f7f** to **#707070**
– @danieljbryson 

As this is a breaking change we may have to consider it for Toolkit 2.0 release
